### DISCUSSION TITLE
how to correctly link notes using tp.date

### DISCUSSION THREAD

**Discussion by fritaz** [SilentVoid13/Templater#431] [Help]
Hey, forgive me if this is a dumb question because I'm completely new to this. I am trying to set up the daily notes and I'm not sure if I'm missing a plugin or I forgot to check off a function within Obsidian.

This is what I see when hovering over the file name in preview mode:

"<%+ tp.date.now("YYYY-MM-DD", 1, tp.file.title, "YYYY-MM-DD") %>" is not created yet. click to create.

then when clicked to create I see:

File name cannot contain any of the following characters * " \ / < > : | ?

![Templaters question](https://user-images.githubusercontent.com/4490589/139541874-b4b066db-4086-4ccb-9eed-31b7294c1706.png)
![file name cannot contain](https://user-images.githubusercontent.com/4490589/139541878-d9f82ad2-3fe9-4750-8f08-7dfa244e62be.png)

So as far as I can tell the function(?)  _tp.date.now("YYYY-MM-DD", 1, tp.file.title, "YYYY-MM-DD") %_  isn't linking to the daily note I've created with the output that is shown in the preview. I'm uncertain on how to rectify this and would love some feedback.

Cheers

**Comment by AB1908** [comment]
Dynamic templater commands can't really link the way you're looking for. Consider using a different approach.
